admin.adoc

```diff
@@ -404,6 +404,70 @@ TBD. User contributions welcome.
 Using the wsgi hasn't been thoroughly tested. If you test, please send feedback
 to the dev list and we can try and fix things.
 
+Running the GRR HTTP Server Behind Apache Reverse Proxy
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
Add a short explanation here about why you would want to do this. Something like:

```
Running apache as a reverse proxy in front of the GRR admin UI is a good way to provide SSL protection for the UI traffic and also integrate with corporate single-sign (if available) on for authentication.
```